modules/comment/comment.service.js

```diff
@@ -1,28 +1,28 @@
-const Comments = require('./comments.model');
+const Comments = require('./comment.model');
 
 class CommentsService {
   getAllComments() {
```

Comment:
Do we need this method at all? What is the reason to get all comments without any connection to any product

Reply:
removed